lib/plugins/tag/include_code.js

```diff
@@ -64,7 +64,7 @@ module.exports = function(ctx) {
       // If the language is not defined, use file extension instead
       lang = lang || pathFn.extname(path).substring(1);
 
-      caption = '<span>' + title + '</span><a href="' + ctx.config.root + codeDir + path + '">view raw</a>';
+      caption = '<span>' + title + '</span><a href="' + pathFn.join(ctx.config.root, codeDir, path) + '">view raw</a>';
```

Comment:
The `join` will generate `\\` on Windows platform, but `\\` is not a valid URI hierarchy separator, see [RFC 1738 Section 2.3](https://www.ietf.org/rfc/rfc1738.txt).

Reply:
```
pathFn.posix.join
```